libs/sdk-ui-dashboard/src/presentation/scheduledEmail/DefaultScheduledEmailDialog/ScheduledMailDialogRenderer/Attachments/ScheduleDropdown.tsx

```diff
@@ -91,15 +100,31 @@ export const ScheduleDropdownComponent: React.FC<IScheduleDropdownProps> = (prop
             overlayZIndex={DEFAULT_DROPDOWN_ZINDEX}
             ignoreClicksOnByClass={[".schedule-dropdown-ignore-click"]}
             renderButton={({ toggleDropdown }) => (
-                <a
-                    className={cx("gd-schedule-dropdown-button", buttonClassName)}
-                    target="_blank"
-                    rel="noopener noreferrer"
-                    onClick={toggleDropdown}
-                >
-                    {iconComponent && <div className="gd-schedule-dropdown-button-icon">{iconComponent}</div>}
-                    {title}
-                </a>
+                <BubbleHoverTrigger showDelay={0} hideDelay={0}>
+                    <a
+                        className={cx(
+                            "gd-schedule-dropdown-button",
+                            buttonClassName,
+                            buttonDisabled && "gd-schedule-dropdown-button-disabled",
+                        )}
+                        target="_blank"
+                        rel="noopener noreferrer"
+                        onClick={() => {
+                            if (buttonDisabled) return;
+                            toggleDropdown();
+                        }}
+                    >
+                        {iconComponent && (
+                            <div className="gd-schedule-dropdown-button-icon">{iconComponent}</div>
+                        )}
+                        {title}
+                    </a>
+                    {buttonDisabled && bubbleMessageId && (
+                        <Bubble className="bubble-primary" alignPoints={ALIGN_POINTS}>
+                            <FormattedMessage id={bubbleMessageId} />
```

Comment:
The translation id here does not need to be passed from parent, right? Let's make it inline if there is no logic and spare the component one prop.